configure

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
-Rscript tools/run_generate_client.R
+${R_HOME}/bin/Rscript tools/run_generate_client.R
```

Comment:
Does `${R_HOME}` need to be `"${R_HOME}"`. The docs seem to suggest it does:

https://cran.r-project.org/doc/manuals/r-devel/R-exts.html#Configure-and-cleanup